th/about.html

```diff
@@ -59,8 +59,9 @@
 				<h3>แหล่งข้อมูล</h3>
 				<ul>
 					<li>ข้อมูลอนิเมะ : <a href="http://myanimelist.net" target="_blank" title="MyAnimeList.net">MyAnimeList.net</a></li>
-					<li>ข้อมูลรูปภาพ : <a href="http://anilist.co" target="_blank" title="MyAnimeList.net">anilist.co</a></li>
-					<li>ข้อมูลแฟนซับ : <a href="https://www.tafasu.com" target="_blank" title="MyAnimeList.net">TAFASU.com</a></li>
+          <li>ข้อมูลรูปภาพ : <a href="http://anilist.co" target="_blank" title="anilist.co">anilist.co</a></li>
+          <li>ข้อมูลแฟนซับ : <a href="https://www.tafasu.com" target="_blank" title="TAFASU.com">TAFASU.com</a></li>
+          <li>ข้อมูลแฟนซับ : <a href="http://www.menome.in.th" target="_blank" title="menome.in.th">menome.in.th</a></li>
```

Comment:
menome.in.th seems a broken link. Is it useful to put their link? A sentence with "some data based from the old menome.in.th" is not sufficient?